ui/app/styles/components/button.scss

```diff
@@ -92,11 +86,23 @@ a[href].button {
   }
 
   &--external-link {
-    > .pds-icon {
+    > svg {
       align-items: flex-end;
     }
   }
 
+  &--add-variable {
+    > span {
+      display: flex;
+      align-items: stretch;
+      margin-left: calc(-1 * 0.25rem);
+
+      svg {
```

Comment:
potentially, also here you have a coupling (less strong than the other one, but still)

one option could be to apply a class to the FlightIcon component and use that one as a CSS selector here (and also in other cases I see where you have used the DOM element as target)